.docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+
+services:
+  firepilot-build:
+    build:
+      context: "${PWD}/.docker"
+      dockerfile: "Dev.Dockerfile"
+    container_name: firepilot-build
+    volumes:
+      - ${PWD}:/build
+    command: "cargo build"
```

Comment:
Hello, why would we want to have a docker compose ? It's only a library

Also, maybe you were thinking on having an homogeneous way to compile the project, and IMO it's handled by the CI. Moreover, building through docker have huge performance issues (going from few seconds of build in bare metal to few minutes in docker)